package.json

```diff
@@ -48,6 +48,7 @@
     "eslint-plugin-jsx-a11y": "6.3.x",
     "eslint-plugin-react": "7.x",
     "eslint-plugin-react-hooks": "4.x",
+    "eslint-plugin-testing-library": "3.9.x",
```

Comment:
We need to update it to version 4 in the near future since it has some decent features that we might need.
But for now, let's just add this version.